controls_pythonbinding.txt

```diff
@@ -1,9 +1,5 @@
-DIR FHEM/bindings
-DIR FHEM/bindings/python
-DIR FHEM/bindings/python/bin
-UPD 2022-08-03_23:37:19 6192 FHEM/10_fhempyServer.pm
-UPD 2022-08-12_18:31:35 4663 FHEM/10_PythonBinding.pm
-UPD 2022-08-13_09:13:18 3253 FHEM/10_PythonModule.pm
-UPD 2022-08-13_20:09:14 2942 FHEM/10_fhempy.pm
-UPD 2022-10-16_21:51:06 19967 FHEM/10_BindingsIo.pm
-UPD 2022-09-16_23:22:44 4504 FHEM/bindings/python/bin/fhempy
```

Comment:
Ich glaube das fehlt noch beim Update des Control Files, oder?
FHEM/bindings/python/bin/fhempy muss nämlich über Controls installiert werden.

Reply:
Jetz bin ich überrascht.

Du sagst also, fhempy wird nicht über pip installiert `pip3 install --upgrade fhempy` sondern es wird ein binary in das FHEM Verzeichnis für die Perl Module geladen?

Reply:
Das ist quasi nur der fhempy Loader. Der führt das pip3 install durch wenn kein fhempy installiert ist oder die Version zu alt ist.
Jetzt sieht es denke ich gut aus 👍